static/test/javascripts/spec/common/commercial/membership-messages.spec.js

```diff
@@ -51,15 +53,20 @@ define([
 
         describe('If user already member', function () {
             beforeEach(function (done) {
-                showMembershipMessages = commercialFeatures.membershipMessages;
+                showMembershipMessages = commercialFeatures.checkWeCanShowMembershipMessages;
                 alreadyVisited = storage.local.get('gu.alreadyVisited');
-                commercialFeatures.membershipMessages = false;
+                window.guardian.adBlockers.onDetect = new Promise(function (resolve) {
+                    resolve({
+                        generic: false,
+                        ffAdblockPlus: false
+                    });
+                });
```

Comment:
I'm a bit confused as to why we set `window.guardian.adBlockers.onDetect` to a promise here, whereas elsewhere this property is set to a function?